### PR TITLE
[#2275] Font sizes were being inherited differently between firefox/chrome with iframe textarea

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2275-embed-iframe-textarea-too-small-font-too-large

### CODE REVIEW

File: css/embed.less [hunk 1]
```diff
@@ -342,6 +342,8 @@ body {
 
   .share-options input, 
   .share-options textarea {
+    font-size: 0.6em;
```
Comment (k88hudson) [inline css/embed.less]: Why use ems?

Comment (mjschranz) [inline css/embed.less]: Mostly just the wrong impression of where ems are better.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/363d1cee84fe31d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/10dc491156b6135/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/363d1cee84fe31d/output.txt

Total script time: 1.97 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/10dc491156b6135/output.txt

Total script time: 1.30 mins

Server is running on http://pr973.butyr.org/templates/basic/
Run the Test Suite: http://pr973.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b4130eca003385f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b5f90c55f4e2b3a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/b4130eca003385f/output.txt

Total script time: 2.83 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b5f90c55f4e2b3a/output.txt

Total script time: 2.40 mins

Server is running on http://pr973.butyr.org/templates/basic/
Run the Test Suite: http://pr973.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5c87a234735e03a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/5c87a234735e03a/output.txt

Total script time: 1.75 mins

**Pull request merged**